devtools/test_dashboard/style.css

```diff
@@ -61,4 +61,5 @@ header span{
 }
 .dashboard-plot{
   margin-bottom: 30px;
+  width: 1100px;
```

Comment:
full window width default width deforms most plots. I think `1000px` is more sensible. Of course this can always be override if desired.